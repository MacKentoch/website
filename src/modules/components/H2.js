import styled from 'styled-components'

const H2 = styled.h2`
  font-size: 26px;
  text-align: center;
  line-height: 1.4;
  font-weight: 300;
  margin: 20px 0;
  @media (min-width: 700px) {
    font-size: 40px;
    text-align: left;
  }
`

export default H2
