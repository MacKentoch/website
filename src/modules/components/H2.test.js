import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import H2 from './H2'

describe('H2', () => {
  it('should render a h2', () => {
    const wrapper = shallow(<H2>Hello</H2>)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
