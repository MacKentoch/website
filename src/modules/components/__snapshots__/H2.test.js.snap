// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`H2 should render a h2 1`] = `
<h2
  className="sc-bdVaJa hhhpXD"
>
  Hello
</h2>
`;
